CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ Inspired by [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 Note: this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.3.1] - Unreleased
```

Comment:
Should this be a major version bump?

Reply:
It was, to `1.3.0`. I yanked that and released this.